x-pack/plugins/elastic_assistant/server/lib/model_evaluator/datasets/esql_dataset.json

```diff
@@ -0,0 +1,42 @@
+  },
+  {
+    "input": "From, employees, I want to sort the documents by salary, and then return 10 results per page, and then see the second page.",
+    "reference": "Pagination is not supported"
+  },
+  {
+    "input": "My, logs data (ECS) is in `logs-*`. Show me a query that gets the average CPU per host, limit it to the top 10 results, in 1m buckets, and only include the last 15m.",
+    "reference": "FROM logs-*\n| WHERE @timestamp >= NOW() - 15 minutes\n| EVAL bucket = DATE_TRUNC(1 minute, @timestamp)\n| STATS avg_cpu = AVG(system.cpu.total.norm.pct) BY bucket, host.name\n| LIMIT 10"
+  },
+  {
+    "input": "I, want to show a list of services with APM data. My data is in `traces-apm*`. I want to show the average transaction duration, the success rate (by dividing event.outcome:failure by event.outcome:failure+success), and total amount of requests. As a time range, select the last 24 hours.",
+    "reference": "FROM traces-apm*\n| WHERE @timestamp >= NOW() - 24 hours\n| EVAL successful = CASE(event.outcome == \"success\", 1, 0),\n  failed = CASE(event.outcome == \"failure\", 1, 0)\n| STATS success_rate = AVG(successful), \n  avg_duration = AVG(transaction.duration), \n  total_requests = COUNT(transaction.id) BY service.name"
+  },
+  {
+    "input": "from, `metricbeat*`, I want to see the percentage of CPU time normalized by the number of CPU cores, broken down by hostname. the fields are system.cpu.user.pct, system.cpu.system.pct, and system.cpu.cores.",
+    "reference": "FROM metricbeat*\n| EVAL cpu_pct_normalized = (system.cpu.user.pct + system.cpu.system.pct) / system.cpu.cores\n| STATS AVG(cpu_pct_normalized) BY host.name"
+  },
+  {
+    "input": "I, want to see a query that does the following: extract the query duration from postgres log messages, and calculate the avg.",
+    "reference": "FROM postgres-logs\n| DISSECT message \"%{} duration: %{query_duration} ms\"\"\n| EVAL query_duration_num = TO_DOUBLE(query_duration)\n| STATS avg_duration = AVG(query_duration_num)"
+  },
+  {
+    "input": "I,'ve got APM data in `metrics-apm`. Filter on `metricset.name:service_destination` and the last 24 hours. Break down by span.destination.service.resource. Each document contains the count of total events (span.destination.service.response_time.count) for that document's interval and the total amount of latency (span.destination.service.response_time.sum.us). A document either contains an aggregate of failed events (event.outcome:success) or failed events (event.outcome:failure). A single document might represent multiple failures or successes, depending on the value of span.destination.service.response_time.count. For each value of span.destination.service.resource, give me the average throughput, latency per request, and failure rate, as a value between 0 and 1.  Just show me the query.",
+    "reference": ""
```

Comment:
consider removing this entry since we don't have an expected result

Reply:
Resolved in [7a4f3b9](https://github.com/elastic/kibana/pull/167220/commits/7a4f3b92e3c8eb0f32f53709196bcc09bac72803) 👍